app/views/dashboard/index.html.haml

```diff
@@ -1,17 +1,18 @@
 - content_for :after_title, " - " + t("application.my_dashboard")
 
-#react_root
+#react_root{data: {default_course_type: ENV["default_course_type"],
```

Comment:
Instead of calling the env variables from the template, let's set those values in DashboardPresenter and call them like other stuff through @pres.